jaxrs/pom.xml

```diff
@@ -123,6 +123,18 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>io.reactiverse</groupId>
+            <artifactId>reactive-contexts-propagators-rxjava1</artifactId>
+            <version>${reactiversecontexts.version}</version>
+            <exclusions>
```

Comment:
Is this because it uses a later version? Perhaps we should consider upping ours instead.